arkouda/pdarraysetops.py

```diff
@@ -142,12 +142,17 @@ def in1d(pda1 : Union[pdarray,Strings,'Categorical'], pda2 : Union[pdarray,Strin
     array([False, True])
     """
     from arkouda.categorical import Categorical as Categorical_
+    if isinstance(pda2, pdarray) or isinstance(pda2, Strings) or isinstance(pda2, Categorical_):
+        from arkouda.dtypes import int_scalars, structDtypeCodes
+        if pda2.size == 0:  # If pda2 is empty, in1d shouldn't do anything.
+            repMsg = generic_msg(cmd="create", args="int64 0")
+            return  create_pdarray(cast(str,repMsg))
```

Comment:
Hmm... this looks to return a zero length pdarray of type int64, but the docs say it should return a boolean array of the same length as pda1.

I think in the case where `pda2` is zero length we should either throw a client-side error or return a False vector of the same length of `pda1`

Reply:
@ezfman thanks for contributing! I would agree with glitch's second recommendation to return an all-False pdarray of the same length as `pda1`. This would be in keeping with the semantics of `in1d`, which essentially says "test each element of `pda1` for membership in the set `pda2`". If `pda2` is the empty set, then all the tests should return False. You can create the all-False pdarray with:

```python
allFalse = zeros(pda2.size, dtype='bool')
```

because "zero" for a bool dtype translates to False.

Reply:
Good to know; I was having some difficulty confirming that data structures were being constructed on the server as opposed to locally.  Looking deeper into the `create_pdarray.py` file has helped clear that up a lot.